web/templates/web/domains/mailshot/partials/detail.html

```diff
@@ -36,14 +36,14 @@
         <div class="row">
           <div class="twelve columns">
             <label class="radio-label">
-              <input type="checkbox" aria-label="Importers and Agents" checked="checked" disabled="disabled">Importers and Agents
+              <input type="checkbox" aria-label="Importers and Agents" disabled="disabled" {{ 'checked="checked"' if object.is_to_importers else ""}}>Importers and Agents
```

Comment:
my measly personal opinion is that the below is clearer but up to you:

```suggestion
              <input type="checkbox" aria-label="Importers and Agents" disabled="disabled" {% if  object.is_to_importers %}checked="checked"{% endif %}>Importers and Agents
```

Reply:
also no need to do `checked="checked"`, the presence of the `checked` attribute is fine

Reply:
I never knew those kind of attributes are called [boolean attributes](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML).

You learn something new every day :)